website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -0,0 +1,247 @@
+
+* **Deprecated annotations, labels and tags are removed for v1.0.0**: For v1, `karpenter.sh/do-not-consolidate` (annotation), `karpenter.sh/do-not-evict
+(annotation)`, and `karpenter.sh/managed-by` (tag) all have support removed.
+The `karpenter.sh/managed-by`, which currently stores the cluster name in its value, is replaced by `eks:eks-cluster-name`, to allow
+for [EKS Pod Identity ABAC policies](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-abac.html).
+
+* **Zap logging removed**: Support for setting the Zap logging config was deprecated in beta and is now removed for v1. View the [Logging Configuration Section of the v1beta1 Migration Guide]({{<ref "../../v0.32/upgrading/v1beta1-migration#logging-configuration-is-no-longer-dynamic" >}}) for more details.
+
+* **metadataOptions could break workloads**: If you have workload pods that are not using `hostNetworking`, the updated default `metadataOptions` could cause your containers to break when you apply new EC2NodeClasses on v1.
+
+* **Support for native Ubuntu AMI selection is dropped**: If you are using Ubuntu, be aware that Karpenter v1 no longer natively supports Ubuntu. To continue using Ubuntu in v1, you can update `amiSelectorTerms` in your EC2NodeClasses to identify Ubuntu as the AMI you want to use. See [reimplement amiFamily](https://github.com/aws/karpenter-provider-aws/pull/6569) for an example. Once you have done that, you can leave the amiFamily as Ubuntu and proceed to upgrading to v1. This will result in the following Transformation:
+   ```yaml
+   version: karpenter.k8s.aws/v1beta1
+   kind: EC2NodeClass
+   spec:
+     amiFamily: Ubuntu
+     amiSelectorTerms:
+     - id: ami-foo
+   ---
+   # Conversion Webhook Output
+   version: karpenter.k8s.aws/v1
+   kind: EC2NodeClass
+   metadata:
+     annotations:
+       compatibility.karpenter.k8s.aws/v1beta1-ubuntu: amiFamily,blockDeviceMappings
+   spec:
+     amiFamily: AL2
+     amiSelectorTerms:
+     - id: ami-foo
+     blockDeviceMappings:
+     - deviceName: 'dev/sda1'
+       rootVolume: true
+       ebs:
+         encrypted: true
+         volumeType: gp3
+         volumeSize: 20Gi
+   ```
+   **NOTE**: If you do not specify `amiSelectorTerms` before upgrading with an Ubuntu AMI, the conversion webhook will fail. If you do this, downgrade to Karpenter `v0.37.1 and add amiSelectorTerms before upgrading again.
+
+* **Migrating userData**: Consider migrating your userData ahead of the v1.0.0 upgrade. See the description of `amiSelectorTerms` below.
+
+### Before upgrading to `1.1.0`
+
+Apply the following changes to your NodePools and EC2NodeClasses, as appropriate, before upgrading them to `v1.1.0` (though okay to make these changes for `1.0.0`)
+
+* **v1beta1 support gone**: In `1.1.0`, v1beta1 is not supported. Migrate all Karpenter yaml files ([NodePools]({{<ref "../concepts/nodepools" >}}), [EC2NodeClasses]({{<ref "../concepts/nodeclasses" >}}), and so on) to v1.
```

Comment:
Can we also call out that rollback to v1beta1 is not supported? Once customers are using `1.1.0`